### PR TITLE
[Fori_loop|While_loop] Add nested test

### PR DESCRIPTION
**Pull request by ManfeiBai**
Add nested test case for while_loop

### CODE REVIEW

**Comment by ManfeiBai** [comment]
since its only add test case, we would not include it in 2.3 but only in nightly, so remove @lsy323's review here too and remove `backport 2.3` label too

File: test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py [hunk 1]
```diff
@@ -55,6 +55,42 @@ def body_fn(init, limit_value):
...
+      two_value = limit_value.clone()
+      return (torch.sub(torch.sub(init, one_value), one_value), two_value)
+
+    init = torch.tensor([10], dtype=torch.int32, device=device)
+    limit_value = torch.tensor([0], dtype=torch.int32, device=device)
+    res = while_loop(cond_fn, body_fn, (init, limit_value))
+    expected = _fake_while_loop(cond_fn, body_fn, (init, limit_value))
+    self.assertEqual(expected, res)
+
+  def test_while_loop_tpu_addition_nested(self):
```
Comment (yeounoh) [inline test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py:76]: let's just add substration, it's more or less the same.

**Review by yeounoh — APPROVED** [review]
LGTM

File: test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py (hunk 1 above)
Comment (ManfeiBai) [reply test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py]: Thanks, agree, updated
